app/assets/stylesheets/comments.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Comments controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```

Comment:
- [ ] It is often considered good practice to try and always take out unused files and remove all comments from files